src/tests/components/forms/FamilyForm.test.tsx

```diff
@@ -0,0 +1,138 @@
+import { configure, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { render, cleanup } from '@testing-library/react'
+import {
+  configMock,
+  mockDocument,
+  mockFamiliesData,
+} from '@/tests/utilsTest/mocks'
+import { FamilyForm } from '@/components/forms/FamilyForm'
+import { TFamily } from '@/interfaces'
+import 'jest-localstorage-mock'
+import { TestWrapper } from '@/tests/utilsTest/render'
+
+const flushPromises = async () =>
+  new Promise((resolve) => process.nextTick(resolve))
+
+// useBlocker only can be used in a router context
+jest.mock('react-router-dom', (): unknown => ({
+  ...jest.requireActual('react-router-dom'),
+  useBlocker: jest.fn(),
+}))
+
+// API calls
+jest.mock('axios', () => {
```

Comment:
I'd be interested to know if we still need this mock?

Reply:
You've been completely right